# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/test/test_text-fragments-create-text-directive.html
# Commit: b1fda3f9af8a
# Full Hash: b1fda3f9af8ae58da0ea4a24078dd75b784e79cf
# Author: Cosmin Sabou <csabou@mozilla.com>
# Date: 2025-09-10 09:37:11
# Regressor Bug: 1979588
# File Overlap Count: 2
# Overlapping Files: dom/base/test/test_text-fragments-create-text-directive.html, dom/base/TextDirectiveCreator.cpp
# Description:
#   Revert "Bug 1979588 - Text Fragments: Correctly deal with long words at end of range for range-based matching. r=dom-core,smaug" for causing crashes (bug 1987845) as part of the Nightly merge operations
#   
#   This reverts commit 7db1bf44de9eb0bcf7b68ff1b5206fc801432ee0.
# ==============================================================================

diff -r 89d6d9a06132 -r b1fda3f9af8a dom/base/test/test_text-fragments-create-text-directive.html
--- a/dom/base/test/test_text-fragments-create-text-directive.html	Wed Sep 10 01:47:49 2025 +0000
+++ b/dom/base/test/test_text-fragments-create-text-directive.html	Wed Sep 10 12:33:12 2025 +0300
@@ -43,8 +43,6 @@
   <div>HereEndsATableAndThisIsHereToMakeSureTheTestResultsAreIndependentOfSurroundingContent</div>
 
   <span id="longWord">1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111</span>
-  <!-- Test case for Bug 1979588: Long URL with unbreakable segment at the end -->
-  <div id="longUrlForBug1979588">https://example.com/test?foo=bar#verylongnonbreakableurlsegmentthatcannotbebrokenintowordsbecauseiturlsegmentsdonotcontainspacesorothertraditionwordbreakingcharacterswhichcausesproblemsintextdirectivecreation</div>
   <p id="loremIpsum">Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source.</p>
   <p>
     <span id="beforeEmptyTextNode">before empty text node</span>
@@ -431,15 +429,6 @@
           textDirective: "text=%7D",
           dontCompareRangeBoundaries: true,
         },
-        {
-          name: "Test for Bug 1979588: Long URL with unbreakable segment should not crash",
-          startContainer: longUrlForBug1979588.firstChild,
-          startOffset: 0,
-          endContainer: longUrlForBug1979588.firstChild,
-          endOffset: longUrlForBug1979588.firstChild.length,
-          content: "https://example.com/test?foo=bar#verylongnonbreakableurlsegmentthatcannotbebrokenintowordsbecauseiturlsegmentsdonotcontainspacesorothertraditionwordbreakingcharacterswhichcausesproblemsintextdirectivecreation",
-          textDirective: "text=https,verylongnonbreakableurlsegmentthatcannotbebrokenintowordsbecauseiturlsegmentsdonotcontainspacesorothertraditionwordbreakingcharacterswhichcausesproblemsintextdirectivecreation",
-        },
       ]) {
         const range = document.createRange();
         range.setStart(testCase.startContainer, testCase.startOffset);
